### PR TITLE
[test] Making the jacobi check generic

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix UES-384

### CODE REVIEW

File: cscs-checks/tools/profiling_and_debugging/notool.py [hunk 1]
```diff
@@ -46,43 +47,44 @@ def __init__(self, lang):
...
         # a scopedict is better than this:
+        self.language = lang
         # if (self.language == 'C++' and
         #    self.current_environ.name == 'PrgEnv-pgi'):
         #    self.omp_versions['PrgEnv-pgi'] = '200805'
-
+        self.maintainers = ['JG', 'MK']
+        self.tags = {'production'}
+        if self.current_system.name in {'dom', 'daint'}:
+            self.post_run = ['module list -t']
```
Comment (victorusu) [inline cscs-checks/tools/profiling_and_debugging/notool.py]: Why are you listing the modules as a post run command and not having any sanity check associated to it? Is there anything we should be aware of? Or it is just to "save" the modules for later check?

Comment (jgphpc) [reply cscs-checks/tools/profiling_and_debugging/notool.py]: https://github.com/eth-cscs/reframe/pull/804/commits/4942a8c52b8428aed33e5f98392ef7c51aabcea0

File: cscs-checks/tools/profiling_and_debugging/notool.py [hunk 2]
```diff
@@ -23,16 +22,18 @@ def __init__(self, lang):
...
         if lang == 'F90':
             self.build_system.max_concurrency = 1
 
         self.num_tasks = 3
         self.num_tasks_per_node = 3
         self.num_cpus_per_task = 4
-        self.num_iterations = 200
+        self.num_tasks_per_core = 1
+        self.use_multithreading = False
+        self.num_iterations = 100
```
[the remark this answers is not in the document]
Comment (jgphpc) [reply cscs-checks/tools/profiling_and_debugging/notool.py]: https://github.com/eth-cscs/reframe/pull/804/commits/4942a8c52b8428aed33e5f98392ef7c51aabcea0

File: cscs-checks/tools/profiling_and_debugging/notool.py (hunk 1 above)
Comment (victorusu) [reply cscs-checks/tools/profiling_and_debugging/notool.py]: Based on our offline discussion, then you just want to save the modules for later check and historical check. In this case, there is no need for additional sanity checks.

**Comment by jgphpc** [comment]
> git diff --name-only origin/master

cscs-checks/tools/profiling_and_debugging/notool.py

I don't think cleaning is needed.

**Comment by teojgo** [comment]
@jg github shows 22 files changed. Can you update with the latest master and repush?

**Pull request closed without merging**